.github/workflows/tests.yml

```diff
@@ -25,5 +25,19 @@ jobs:
       - name: Check formatting
         run: pipenv run black --diff --check .
 
+      - name: Run lints
+        run: pipenv run pylint idunn
+
+      # Because of the "hacky" nature of mocks we will disable a few more lints
+      # for tests.
+      - name: Run lints on tests
+        run: >-
+          pipenv run pylint tests --disable="import-outside-toplevel,
+                                             line-too-long,
+                                             protected-access,
+                                             redefined-outer-name,
+                                             unused-argument,
+                                             unused-import"
```

Comment:
How should be run these lint rules locally ?  
Maybe we could define a dedicated script to run these `pylint` commands ? 
Or add a Pipenv "custom script": https://pipenv.pypa.io/en/latest/advanced/#custom-script-shortcuts

Reply:
I like this custom script option!

I would have felt bad about adding a random shell script just for this ^^'